### PR TITLE
[WIP] preparation for fdm test

### PR DESCRIPTION
**Pull request by floweisshardt**
Do not merge yet. needed for test rigs

needed for https://github.com/ipa320/cob_hardware_test/pull/10

### CODE REVIEW

**Comment by fmessmer** [comment]
please remove the mimic related stuff to a separate PR!

**Comment by fmessmer** [comment]
@ipa-fmw please separate the mimic part into a standalone PR

File: cob_bringup/robots/cob4-7.xml [hunk 1]
```diff
@@ -244,9 +244,8 @@
 			</include>
 		</group>
 
-		<include file="$(find cob_bringup)/drivers/mimic.launch">
-			<arg unless="$(arg sim)" name="rotation" value="270"/>
-		</include>
+		<include file="$(find cob_bringup)/drivers/mimic.launch"/>
```
Comment (NadiaHG) [inline cob_bringup/robots/cob4-7.xml]: did you change the head pc display configuration? why is the rotation not needed?

**Comment by fmessmer** [comment]
I rebased to get rid of the mimic changes

what else needs to be done to finalize?
I guess removing the parts uploading parameters will break the driver for the robots...

File: cob_bringup/drivers/canopen_402.launch [hunk 2]
```diff
@@ -18,7 +18,6 @@
 
 	<!-- start canopen node -->
 	<node ns="$(arg component_name)" pkg="canopen_motor_node" type="canopen_motor_node" name="driver" output="screen">
-		<rosparam command="load" file="$(arg pkg_hardware_config)/robots/$(arg robot)/config/$(arg component_name)_driver.yaml"/>
```
Comment (fmessmer) [inline cob_bringup/drivers/canopen_402.launch:21]: I guess removing the parts uploading parameters will break the driver for the robots...

**Review by fmessmer — CHANGES_REQUESTED** [review]
block PR for verification

**Comment by fmessmer** [comment]
@ipa-fmw 
with quite some new cob4_bases coming up...what do we need to finalize the fdm test site (hw-/sw-wise?)

**Comment by floweisshardt** [comment]
finalizing this should be done by end of August, so that we can test the FDMs which will be ready in beginning of september

... just lacking time to finalize this at the moment

**Comment by fmessmer** [comment]
@ipa-fmw
not needed for ipa320/cob_hardware_test#11
can be  closed

**Pull request closed without merging**